Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "AtomsCalculators"
 uuid = "a3e0e189-c65a-42c1-833c-339540406eb1"
 authors = ["JuliaMolSim contributors"]
-version = "0.1.1"
+version = "0.1.2-DEV"
```

Comment:
What's the rationale here? I only modify this to bump versions (you release after pretty much every merge anyway)

Reply:
This is to help other packages to limit to v0.1.2 when it has not been released yet. In this case EmpiricalPotentials can implement Hessian and add compat entry to v0.1.2.

Secondly, I plan to make PR for parameter estimations also and get it to v0.1.2 too.

Thirdly, adding -DEV to version is standard Julia way to inform that you are on a dev version. You don't need to use it, but it can be used to help in cases like above.

Reply:
Ok, thanks for the clarification.